### PR TITLE
add editor title button

### LINKED ISSUE #2: Preview button, like how markdown works
It would be great if open tabs with the `ansi` language had a little preview in new tab button, like how markdown works:

image

Is this possible?

### PR DESCRIPTION
**Pull request by iliazeus**
Resolves #2.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,9 +35,24 @@
       {
         "command": "iliazeus.vscode-ansi.showPretty",
         "title": "Show as ANSI",
-        "enablement": "editorActive"
+        "icon": "$(open-preview)"
+      },
+      {
+        "command": "iliazeus.vscode-ansi.showPrettyToSide",
+        "title": "Show as ANSI to the Side",
```
Comment (jaydenseric) [inline package.json:42]: Should the language more closely match how markdown works?

```suggestion
        "title": "Open Preview to the Side",
```

**Review by jaydenseric — COMMENTED** [review]
The commands could also be updated for better consistency with how markdown works:

Screen Shot 2020-12-04 at 10 14 53 pm

Also, does the preview tab start with the word `Preview ` to distinguish it from the normal editor tab?

File: package.json (hunk 1 above)
Comment (iliazeus) [reply package.json:42]: It makes sense, I guess. I didn't really like the word "preview" initially, but I'm not a native speaker.

**Comment by iliazeus** [comment]
> Also, does the preview tab start with the word Preview to distinguish it from the normal editor tab?

Since I try to use VS Code's built-in read-only editors for display, I'm not sure if it will be possible. But I will look into it.

File: package.json (hunk 1 above)
Comment (iliazeus) [reply package.json:42]: Addressed these issues in 8706dc2.

**Pull request merged**